src/components/Firestore/index.test.tsx

```diff
@@ -91,17 +94,27 @@ describe('Firestore', () => {
 
   it('triggers clearing all data', async () => {
     confirm.mockResolvedValueOnce(true);
-    const nukeSpy = jest.fn();
+    let nukeResolve;
+    const nukeSpy = jest
+      .fn()
+      .mockImplementation(
+        () => new Promise(resolve => (nukeResolve = resolve))
```

Comment:
Please also extract this `Promise` out to alongside nukeResolve so we can `await nukeResolve` instead of `await wait()` later.